README.md

```diff
@@ -53,3 +53,22 @@ jobs:
           WG_DECISION_URL: "https://lists.w3.org/Archives/Public/public-webapps/2014JulSep/0627.html"
           CC: "foo@bar.com,bar@foo.com"
 ```
+
+#### `WG_DECISION_URL`
```

Comment:
Maybe mention above that the URLs are below, and delete the WebAppsWG one?

Reply:
I think example should show example instead of explaining inputs?

Reply:
ok, yeah